### ISSUE TITLE
Deprecation POW

### ISSUE THREAD

**Issue by mreishus** [mreishus/spades#154]
```
Generated pow app
==> spades
Compiling 69 files (.ex)
warning: Pow.Plug.clear_authenticated_user/1 is deprecated. Use `delete/1` instead
  lib/spades_web/controllers/api/v1/session_controller.ex:57: SpadesWeb.API.V1.SessionController.delete/2
```

**Comment by mreishus** [comment]
Also, the API guide has been updated. https://github.com/danschultzer/pow/blob/master/guides/api.md

**Comment by mreishus** [comment]
Fixed in 1213ce2
